docs/development/releasing/index.md

```diff
@@ -5,13 +5,20 @@
 ### 1. Build and test locally:
 
 ```console
-$ DEMO=r-d-r-roofit EXCLUDE_COMPONENTS=r-ui,r-m-broker make ci
+$ reana-dev run-ci \
+    --admin-email john.doe@example.org \
+    --admin-password 123456 \
+    --component r-d-r-roofit \
+    --exclude-components=r-ui,r-m-broker
 ```
 
 ### 2. Build and test using the images to be released:
 
-```yaml
-$ GITHUB_USER=reanahub BUILD_TYPE=release make ci
+```console
+$ reana-dev run-ci \
+    --admin-email john.doe@example.org \
+    --admin-password 123456 \
+    --mode releasehelm
```

Comment:
This is incorrect, the `releasehelm` mode is installing  REANA from released Helm charts.

Also, `run-ci` would build Docker images with included shared modules, one has to use `git-submodule --delete` and `docker-build` commands.

I think we need to revamp the releasing section quite a bit.

For this PR, perhaps just leave visible FIXME note, and we shall update it separately.

Reply:
Ok. I will leave a `# FIXME` comment for now.